guides/common/modules/proc_configuring-project-server-as-red-hat-insights-client.adoc

```diff
@@ -13,23 +12,22 @@ For more information about Red{nbsp}Hat Insights, see https://access.redhat.com/
 
 To maintain your {ProjectServer}, and improve your ability to monitor and diagnose problems you might have with {Project}, install Red{nbsp}Hat Insights on {ProjectServer} and register {ProjectServer} with Red{nbsp}Hat Insights.
 
-.Scheduling insights-client
-
-Note that you can change the default schedule for running `insights-client` by configuring `insights-client.timer` on {Project}.
-For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
-
 .Procedure
-
-. To install Red{nbsp}Hat Insights on {ProjectServer}, enter the following command:
+. Register {ProjectServer} with Red{nbsp}Hat Insights:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {project-package-install} insights-client
+# {foreman-installer} --register-with-insights
 ----
-+
-. To register {ProjectServer} with Red{nbsp}Hat Insights, enter the following command:
-+
+For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-configuring-insights-client#proc-registering-system-insights_insights-cg-configuring-client[Registering your system with Red Hat Insights] in the _Client Configuration Guide for Red Hat Insights_. 
+. Optional: You can change the default schedule for running `insights-client` by configuring `insights-client.timer` on {Project}.
+For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
+. Optional: To unregister the system with Red{nbsp}Hat Insights:
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --register-with-insights
+# insights-client --unregister
 ----
+For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-configuring-insights-client#proc-unregistering-system-insights_insights-cg-configuring-client[Unregistering your system with Red Hat Insights] in the _Client Configuration Guide for Red Hat Insights_.
```

Comment:
On second thought, I would consider making this a NOTE admonition instead of a procedure step. Or a next step under the allowed ".Next step" heading? What do you think?

Reply:
Since next steps is supposed to be meant for steps that the user might find useful to refer to after registration, it probably makes for a better candidate than adding an optional step in the procedure.